src/options.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path');
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Use a standard "import" statement instead of "require(...)". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AS3533)